File: PYTHON PROGRAMMING/Test66.py
# 66. PERFECT SQUARE NUMBER
from math import *
def PerfectSquare(num):
    sqrtnum = floor(sqrt(num))
    flag = (sqrtnum * sqrtnum == num)
    if flag:
        print(f"{num} Is A Perfect Square Number.")
    else:
        print(f"{num} Is Not A Perfect Number.")
PerfectSquare(36)
PerfectSquare(15)
PerfectSquare(36)

File: JAVASCRIPT PROGRAMMING/Test66.js
"use strict";
// 66. PALINDROME NUMBER
const Palindrome = (num) => {
    let NUM = num, rev = 0;
    while (num != 0) {
        let rem = num % 10;
        rev = (rev * 10) + rem;
        num = Math.floor(num / 10);
    }
    let res = (NUM == rev) ?
        `${NUM} Is A Palindrome Number.` : `${NUM} Is Not A Palindrome Number.`;
    console.log(res);
};
Palindrome(121);
Palindrome(153);